sensor_msgs/NavSatFix point
---
float64 reward